next.clayui.com/content/docs/components/sticker.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Sticker"
+description: "Stickers are a visual way to quickly identify content in a different way than badges and labels."
+packageNpm: "@clayui/sticker"
+packageVersion: "3.0.0"
+packageStatus: "Beta"
+---
+
+import {Sticker, StickerColors, StickerSizes, StickerPositions, StickerOutside, StickerOverlay, StickerUserIcon} from '../../../src/components/clay/Sticker';
+
+<div class="nav-toc">
+
+- [Variants and Best Pratices](#variants-and-best-pratices)
```

Comment:
hey @diegonvs I would say the same thing here that I mentioned in the button documentation. https://github.com/liferay/clay/pull/2062#discussion_r294529739